tests/FixerFactoryTest.php

```diff
@@ -380,15 +379,38 @@ public function testFixerDefinitions(FixerInterface $fixer)
             $this->markTestIncomplete('ShortFixerDefinition does not contains all needed information.');
         }
 
-        $this->assertNotEmpty($definition->getCodeSamples(), 'Code samples are required.');
+        $samples = $definition->getCodeSamples();
+        $this->assertInternalType('array', $samples, 'Code samples must be an array.');
```

Comment:
return type of `getCodeSamples` is `array`, always.
not sure that we need to check if one is violating the interface... we don't do it anywhere else